src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -756,6 +803,8 @@ define([
          * Starts the resolution dropdown for the preview
          */
         startPreviewResolutionDropdown: function() {
+            var instanceName = 'resolutionsDropdown';
```

Comment:
i think in some stage of development i used it twice in this function but i will remove it!